pkg/apis/core/validation/validation.go

```diff
@@ -2943,6 +2945,18 @@ func validateExecAction(exec *core.ExecAction, fldPath *field.Path) field.ErrorL
 	return allErrors
 }
 
+func validateSleepAction(sleep *core.SleepAction, terminationGracePeriodSeconds *int64, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	gracePeriodSeconds := int64(30)
+	if terminationGracePeriodSeconds != nil {
+		gracePeriodSeconds = *terminationGracePeriodSeconds
+	}
+	if sleep.Seconds <= 0 || sleep.Seconds > gracePeriodSeconds {
+		allErrors = append(allErrors, field.Invalid(fldPath, sleep.Seconds, validation.InclusiveRangeError(1, int(gracePeriodSeconds))))
```

Comment:
I think it would be better to say:

```
"must be greater than 0"
```

and 

```
"may not be longer than `terminationGracePeriodSeconds` (%d)"
```